client/src/actions/dataActions.js

```diff
@@ -0,0 +1,9 @@
+// @flow
```

Comment:
Why is this file called `dataActions`? Seems like a pretty generic name, woudln't `collabActions` be better?

Reply:
It is generic but I tried to go with existing naming conventions. It seems that names of data providers in `dataProviders` are inferred from [API](https://verejne.digital/test.html) (from the name of "Application" the endpoint belongs to, in this case, the name is `data`). And the names of actions are inferred from the name of data providers...

I can rename it if you want. Let me know...

Reply:
I'm really not sure here. It makes sense to have it consistent with api endpoints, but I really wouldn't know what to expect from this file. I am slightly leaning towards renaming. But no pressure, the final decision is up to you.